binding_generator.py

```diff
@@ -28,7 +28,7 @@ def generate_bindings(path):
         source_file.write(impl)
     
     
-    icall_header_file = open("src/__icalls.hpp", "w+")
+    icall_header_file = open("include/__icalls.hpp", "w+")
```

Comment:
I put it in src on purpose since it's not meant to be user facing

Reply:
I can undo it but I thought it looked ugly :P its hardly an effective way to hide it the _____ should be telling enough